### ISSUE TITLE
Only hero image is able to be changed to avif

### ISSUE THREAD

**Issue by theapplegates** [wangonya/flexible-gatsby#14]
I added the following code in the blog-list.js and the blog-post.js

`gatsbyImageData(placeholder: BLURRED, layout: FULL_WIDTH, formats: [AUTO, AVIF, WEBP])`

After running Gatsby develop and build I found only the hero image was avif.

The image on the island in this post is jpg and should be avif.
https://quirky-sammet-031486.netlify.app/how-i-rest/

Note the surfer in the ocean at the top of the post is avif

Any ideas what I am doing wrong? I think it's obvious I am not a developer.

Thanks
Paul

**Comment by wangonya** [comment]
Hey @theapplegates, sorry for the late response on this. Is the repo where you did this public? If so, please share it. Looking at the code will help me figure this out faster.

**Comment by theapplegates** [comment]
Thanks for looking at it. Here is the repo, https://github.com/theapplegates/upgraded-garbanzo-gat-flex

I changed two files
src/templates/blog-post.js 
and
src/templates/blog-list.js

Line 122 in blog-list.js I added the AVIF format option
`gatsbyImageData(placeholder: BLURRED, layout: FULL_WIDTH, formats: [AUTO, AVIF, WEBP])`
I thought this would allow avif files in the browsers that support them.
Only the hero image was changed when viewed in Chrome, Firefox, or Opera.
A image in the body of the post was kept as webp, not avif.

I changed line 86 in blog-post.js

`gatsbyImageData(placeholder: BLURRED, layout: FULL_WIDTH, formats: [AUTO, AVIF, WEBP])`

Same issue. I did not know which one controlled the image format, but I guessed it did since I saw `gatsbyImageData`


I am not a developer but the hero image is displayed as avif, so I thought the pic in the body of a post would change too.

Thanks for the help
Paul
